mmv1/third_party/terraform/resources/resource_sql_user.go

```diff
@@ -84,6 +84,18 @@ func resourceSqlUser() *schema.Resource {
 				ValidateFunc: validation.StringInSlice([]string{"BUILT_IN", "CLOUD_IAM_USER", "CLOUD_IAM_SERVICE_ACCOUNT", ""}, false),
 			},
 
+			"disabled": {
```

Comment:
These values are nested in a block in the underlying API, can we represent them the same way here? w/o direct object support, they'll be represented as a `TypeList` with `MaxItems` set to `1`.

Reply:
I dont have a preference here - i'm not the original author, just trying to get the ball rolling since it had stagnated in the downstream repo, and i have need of this functionality.

From an API perspective, i'm not sure why these are in a nested object, as they are seemingly unrelated values.  Is there a goal of having the terraform resource match the underlying API exactly?

Reply:
Yeah, we'd like to match the API where possible (i.e. unless the original API representation is actively harmful)

Reply:
Ok - change pushed to make this nested